mappings/net/minecraft/sortme/EightHorizontalDirection.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS ez net/minecraft/sortme/EightHorizontalDirection
+	FIELD a NORTH Lez;
+	FIELD b NORTHEAST Lez;
+	FIELD c EAST Lez;
```

Comment:
again, not these,fields, we have a way to extract this data from enums

Reply:
Hmm, I did not have a chance to read their compiled constructors. Their decompiled constructors only takes an array of facings for argument, which I named these constants from.

Reply:
An automatic way. There's a tool which appends the heuristics information to Tiny-mappings and it will be integrated with Enigma soon-ish.